<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgMirrors</class>
 <widget class="QDialog" name="CQDlgMirrors">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mirror Properties</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>411</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Main properties</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QCheckBox" name="qqEnabled">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Enabled</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>85</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Mirror</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QRadioButton" name="qqIsMirror">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Mirror function</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqReflectance">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>25</y>
      <width>76</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>25</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Mirror reflectance</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Clipping plane</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QRadioButton" name="qqIsClippingPlane">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>371</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Clipping plane function, entity to clip:</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="qqEntityCombo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>45</y>
      <width>261</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="qqDisableAllMirrors">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Disable all mirrors</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="qqDisableAllClippingPlanes">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>201</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Disable all clipping planes</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Visual properties</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QToolButton" name="qqColor">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>25</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Adjust color</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Height [m]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqHeight">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>45</y>
      <width>76</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqWidth">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>25</y>
      <width>76</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Width [m]</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqDisableAllMirrors</tabstop>
  <tabstop>qqDisableAllClippingPlanes</tabstop>
  <tabstop>qqEnabled</tabstop>
  <tabstop>qqIsMirror</tabstop>
  <tabstop>qqReflectance</tabstop>
  <tabstop>qqIsClippingPlane</tabstop>
  <tabstop>qqEntityCombo</tabstop>
  <tabstop>qqWidth</tabstop>
  <tabstop>qqHeight</tabstop>
  <tabstop>qqColor</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
